### DISCUSSION TITLE
Data Questions

### DISCUSSION THREAD

**Discussion by lbarbula** [openstates/issues#826] [General]
First of all, I have been exploring this API quite a bit in the last couple days and really appreciate the effort to build and maintain it! Secondly, looking to integrate some of this data and maybe the api into a project and had a couple questions regarding some data that I would need that I can't currently seem to get to return regardless of query parameters.

It would appear to me that no matter what combination of parameters, id, name, jurisdiction etc. that I try, I can't get any executives back at a federal level. I can get municipal and state execs, but not national. As an example
`  'https://v3.openstates.org/people?jurisdiction=ocd-jurisdiction%2Fcountry%3Aus%2Fgovernment&include=links&include=sources&include=offices&page=1&per_page=50&apikey={obfuscated}' \
`

^^I would assume that would get the executives, but it doesn't seem to. I also tried the id given by the jurisdictions API call as well without luck. Thanks again

**Comment by johnseekins** [comment]
Hi @lbarbula . It looks federal legislator details are manually added to OpenStates here: https://github.com/openstates/people and we just don't have executives added at a federal level. We'd certainly welcome a contribution, though!
